core/src/cluster_info.rs

```diff
@@ -1112,6 +1112,7 @@ impl ClusterInfo {
             .unwrap()
     }
 
+    #[allow(clippy::cognitive_complexity)]
```

Comment:
Why are all these `#[allow(clippy::cognitive_complexity)]` directives needed when you didn't modify any of the functions you added them to?

Reply:
probably because macros cause some different compiler path to check files.